components/script/dom/bindings/codegen/parser/dataview.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/components/script/dom/bindings/codegen/parser/WebIDL.py b/components/script/dom/bindings/codegen/parser/WebIDL.py
+index 59ec481c97..7348db73cb 100644
+--- a/components/script/dom/bindings/codegen/parser/WebIDL.py
++++ b/components/script/dom/bindings/codegen/parser/WebIDL.py
+@@ -3131,7 +3131,7 @@ class IDLBuiltinType(IDLType):
+                 self.isSharedArrayBuffer() or
+                 self.isTypedArray() or
+                 self.isReadableStream() or
+-                seld.isDataView())
++                self.isDataView())
```

Comment:
This patch file does not contain all the changes on your PR; can you regenerate it?

Reply:
The git diff of this file isnt returning anything is there any other way or workaround to do so?

Reply:
You need to diff the previous commit with the current commit.